src/tox/Network/Tox.lhs

```diff
@@ -26,10 +26,76 @@ lengths (e.g.  \texttt{4 | 16}), or an inclusive range (e.g. \texttt{[0,
 100]}).  Open ranges are denoted \texttt{[n,]} to mean a minimum length of
 \texttt{n} with no specified maximum length.
 
+\section{TODO: Goals and threat model}
+
+This section should give an idea on what are the goals and non-goals of Tox, so
+that reader
+
+\begin{itemize}
+  \item understands what problems Tox intends to solve
+  \item can validate if they are addresed by this specification
+  \item can make better tradeoffs and decisions in his own reimplementation of the
+    protocol
+\end{itemize}
+
+(TODO: this is just a placeholder; some more technical description should be
+given)
+
+What Tox Does:
+
+\begin{itemize}
+  \item Authentication: user's address is its public key, thus "adding friend"
+    actually means verifying key (false in case of toxme?); drawback is that
```

Comment:
@aaannndddyyy where would you draw the line? Should you also mention the issues with TOR, and with HTTPS in general, or just the toxme implementation? How about when you roll your own toxme? 

My point was, if you're trying to be fancy with 3rd party extensions, that decrease security at the cost of convenience, I don't think that's something we should concern ourselves with. Explaining it to client developers, sure, but the spec isn't for client devs.

(I've also updated my original comment with and additional line to make my original comment more clear.